view/theme/vier/mobile.css

```
@@ -1,4 +1,4 @@
-aside, header, #nav-search-box, #nav-admin-link, #nav-messages-linkmenu,  #activitiy-by-date-tab, #shared-links-tab, #nav-logout-link,
+aside, header, #nav-search-box, #nav-admin-link, #nav-messages-linkmenu,  #activity-by-date-tab, #shared-links-tab, #nav-logout-link,
```

Some of the IDs and classes (such as this one) got removed over time. This is harmless and should be fixed up in another PR (ideally with automated tooling as well)

---

I've been looking into all the renamed classes and ids, this one can go.
```suggestion
aside, header, #nav-search-box, #nav-admin-link, #nav-messages-linkmenu, #shared-links-tab, #nav-logout-link,
```